helper/resource/plugin.go

```diff
@@ -158,15 +158,15 @@ func runProviderCommandSavedPlan(ctx context.Context, t testing.T, wd *plugintes
 		plan, err = wd.SavedPlan(ctx)
 		return err
 	}
-	err := runProviderCommand(ctx, t, fn, wd, factories)
+	err := runProviderCommand(ctx, t, wd, factories, fn)
 	if err != nil {
 		return nil, err
 	}
 
 	return plan, nil
 }
 
-func runProviderCommand(ctx context.Context, t testing.T, f func() error, wd *plugintest.WorkingDir, factories *providerFactories) error {
+func runProviderCommand(ctx context.Context, t testing.T, wd *plugintest.WorkingDir, factories *providerFactories, f func() error) error {
```

Comment:
Moved the function to the end of the argument list, so that function calls are a bit nicer to read.